### DISCUSSION TITLE
Is is possible to have nexted cards or mutliple answers in the same card?

### DISCUSSION THREAD

**Discussion by hscissors** [st3v3nmw/obsidian-spaced-repetition#883] [Q&A] [answered]
I have situation when learning Japanese vocab where I'd like to have a English -> Japanese card, but also on the Japanese side, have kanji and hiragana. So for example:

Cat
?
猫
ねこ //<-- This I'd like to have hidden when the card is first "flipped"

Is that possible?

**Comment by ronzulu** [comment]
Hi @hscissors 

At the moment when the card first appears you only see

`Cat`

Then when you click on answer, you see

```
Cat
----
猫
ねこ 
```

What are you wanting exactly?  (You wrote 'This I'd like to have hidden when the card is first "flipped"' only on the `ねこ`). Do you want that when the answer is first shown, you only see:

```
Cat
----
猫
```

And then later you want the full answer?
```
Cat
----
猫
ねこ 
```

Should it display the full answer after a certain time delay after the partial answer? Or some other mechanism?

Cheers
Ronny

**Reply by hscissors** [reply]
Yeah like the second one, where on flip, it shows 猫, and then maybe with a special button or separate tap (like a spoiler on discord) it reveals the second part.

**Comment by ronzulu** [comment, marked as answer]
There is no current functionality within the plug-in that would support this.

Perhaps this could be achieved using some clever css/HTML. The aim being writing a card as:
```
Cat
----
猫
<div class="hideUntilTapped">ねこ </div>
```

The idea being that when the answer is first shown it displays as something like:
```
Cat
----
猫
Click here to see more
```

Unfortunately I only have basic css/HTML skills, and can't assist further.

Cheers
Ronny

**Reply by hscissors** [reply]
Oh that's a great idea. Let me see if that works and I can make a template out of it

**Comment by hscissors** [comment]
@ronzulu That worked great on desktop. Doesn't seem to work at all on mobile (Android), but I'll keep tinkering with it. Thanks!
